spacewalk/setup/bin/spacewalk-setup

```diff
@@ -1233,6 +1236,16 @@ sub setup_users_and_groups {
     return 1;
 }
 
+sub setup_websockify {
```

Comment:
This creates the file when calling SUMA setup, i.e. on the 1st installation. We should also think how to create this file on existing systems after SUMA update. There aren't many options and it's hard to say where this should fit:
*  One of the candidates could be `UpgradeCommand.java` which runs on tomcat start,
* another one could be a `%post` script in rpm,
* another alternative could be executing a script on `mgr-websockify` service start.

Reply:
in fact, after some deeper investigations this is a remnant of the version before JWT. I can just delete that config file thanks to JWT.

Reply:
Very good, keeping that file consistent would be another worry that we  don't have now :+1: